Sources/SnapshotTesting/Snapshotting/SwiftUIView.swift

```diff
@@ -28,11 +28,13 @@ extension Snapshotting where Value: SwiftUI.View, Format == UIImage {
   /// - Parameters:
   ///   - drawHierarchyInKeyWindow: Utilize the simulator's key window in order to render `UIAppearance` and `UIVisualEffect`s. This option requires a host application for your tests and will _not_ work for framework test targets.
   ///   - precision: The percentage of pixels that must match.
+  ///   - subpixelThreshold: The byte-value threshold at which two subpixels are considered different.
   ///   - size: A view size override.
   ///   - traits: A trait collection override.
   public static func image(
     drawHierarchyInKeyWindow: Bool = false,
     precision: Float = 1,
+    subpixelThreshold: UInt8 = 0,
```

Comment:
Would it maybe make sense the default the threshold to a value which will avoid the M1 vs Intel issue out of the box (ex - 5)?

Reply:
I haven't seen any _"one"_  value that would make sense. Sometimes, with corner-rounding for instance, the difference is `<= 2`, while with images it seems to typically be `<= 1`. With shadows and rounded corners, it may be as high as `<= 5`. I think all of these values are anecdotal though, and that the user should have a fully explicit relationship with this value. After all, this **is** a hacky workaround, and it's probably a good idea to not enable it by default.

Reply:
YeH, makes sense, I was just hoping there was a magic value haha.